cpg_workflows/metamist.py

```diff
@@ -55,6 +55,40 @@
         """,
 )
 
+GET_SEQUENCING_GROUPS_BY_COHORT_QUERY = gql(
+    """
+    query SGByCohortQuery($cohort_id: String!) {
+        cohorts(id: {eq: $cohort_id}) {
+            sequencingGroups {
+                id
+                meta
+                platform
+                technology
+                type
+                sample {
+                    project {
+                        name
+                    }
+                    externalId
+                    participant {
+                        id
+                        externalId
+                        reportedSex
+                        meta
```

Comment:
I recently made a change [here](https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/metamist.py#L42) and in inputs.py [here](https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/inputs.py#L42-L43) to facilitate grabbing phenotypes for use in Exomiser. Can we chuck those back in?

both on line 42, how cool is that

Reply:
https://github.com/populationgenomics/production-pipelines/commit/001e92e4d2b8c3b6c6dc39052c90953c9f1e4d53 was the PR, most of the changes were in Exomiser, a little bit in metamist/inputs and corresponding changes to get the tests to pass